connection/connection_params.ts

```diff
@@ -107,7 +114,24 @@ export type ClientControls = {
    * - `strict` : deno-postgres parses the data into JS objects, and if a parser is not implemented, it throws an error
    * - `raw` : the data is returned as Uint8Array
    */
-  decode_strategy?: "string" | "auto";
+  decode_strategy?: DecodeStrategy;
```

Comment:
```suggestion
  decodeStrategy?: DecodeStrategy;
```
The standard JS casing convention is camelCase.